spec/routing/bulletins_routing_spec.rb

```diff
@@ -20,9 +18,9 @@
 
     it 'routes to bulletins#sunday' do
       expect(show_sunday_bulletin).
-          to route_to(controller: 'basechurch/v1/bulletins',
-                      action: 'sunday',
-                      format: 'json')
+        to route_to(controller: 'v1/bulletins',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.